health-api/src/main/java/com/netflix/runtime/health/api/HealthCheckAggregator.java

```diff
@@ -18,7 +18,22 @@
 import java.util.concurrent.CompletableFuture;
 
 /**
+ * The HealthCheckAggregator wraps all registered implementations of {@link HealthIndicator} and is responsible
```

Comment:
'all registered implementations' seems to specific.  Consider generalizing this without making assumption about where the HealthIndicators came from.